README.md

```diff
@@ -92,22 +92,18 @@ annotations:
 
 ## Build <a name="build"></a>
 
-To exclude `jsx.js` to the build add `jsx` to the `stubModules` array and add
-it's dependencies (JSXTransformer, text) to the `exclude` list in the `modules`
-field of the `build.js`.
-
-Add `"react"` if you want it to be in it's own build file.
+If you are using r.js to optimize your application for production, `jsx` and `JSXTransformer` to the `stubModules` array field of the `build.js`.
```

Comment:
> , `jsx` and `JSXTransformer` to the `stubModules` array field of the `build.js`.

Seems like some verb is missing, e.g.

    , add `jsx` and `JSXTransformer` to the `stubModules` array field of the `build.js`.